CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 - **Breaking** Removed `EventLoopExtRunReturn` / `run_return` in favor of `EventLoopExtPumpEvents` / `pump_events` and `EventLoopExtRunOnDemand` / `run_ondemand` ([#2767](https://github.com/rust-windowing/winit/pull/2767))
 - `RedrawRequested` is no longer guaranteed to be emitted after `MainEventsCleared`, it is now platform-specific when the event is emitted after being requested via `redraw_request()`.
   - On Windows, `RedrawRequested` is now driven by `WM_PAINT` messages which are requested via `redraw_request()`
+- **Breaking** `LoopDestroyed` renamed to `LoopExiting` ([#2900](https://github.com/rust-windowing/winit/issues/2900))
```

Comment:
In general we don't link to the PR changing it things.

Would suggest to drop the links, they are redundant and easily obtainable with git blame.

Reply:
surely there's no harm in making more context / information easily accessible from the changelog?

Reply:
it's just overly verbose for what it is, the context is already there with `blame`.

Reply:
it's not verbose when it's rendered though.

I really appreciate when I see project changelogs that link changes back to issues / PRs and find that useful.

Tracking down context with git blame can be a faff, and inevitably at some point something will be re-formatted and it becomes harder to track  context down via blame.